front/src/modules/ui/object/record-table/components/RecordTableHeader.tsx

```diff
@@ -71,10 +75,10 @@ const StyledColumnHeadContainer = styled.div`
   z-index: 1;
 `;
 
-const StyledPlusIconHeaderCell = styled.th`
+const StyledPlusIconHeaderCell = styled.th<{ isSticky: boolean }>`
   ${({ theme }) => {
     return `
-  &:hover {
+  &:hover > div {
```

Comment:
Making the entire button transparent meant you could see content below the button on hover in the case that it was sticky and appeared on top of other content. Here, we instead make the div inside the button, transparent, since the button has a solid background, you can't see the content below.

Reply:
I am not sure to understand why the button would be on top of another content below. It should always be in its own column